net/core/net_namespace.c

```diff
@@ -261,7 +261,7 @@ struct net *get_net_ns_by_id(struct net *net, int id)
 	spin_lock_irqsave(&net->nsid_lock, flags);
 	peer = idr_find(&net->netns_ids, id);
 	if (peer)
-		peer = maybe_get_net(peer);
```

Comment:
I would personally discard this change as whitespace changes result in more conflicts with stable.